README.md

```
@@ -95,17 +98,20 @@ CONVEX_DEPLOYMENT=your_convex_deployment_or_dev_if_local
 ### Installation
 
 1. Clone the repository
+
 ```bash
 git clone https://github.com/yourusername/project-haribon.git
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)